packages/apps/postgres/README.md

```diff
@@ -66,44 +66,57 @@ See:
-| `size`            | Persistent Volume size                                                                                                                  | `10Gi`  |
-| `storageClass`    | StorageClass used to store the data                                                                                                     | `""`    |
-| `external`        | Enable external access from outside the cluster                                                                                         | `false` |
-
-### Application-specific parameters
-
-| Name                                    | Description                                                                                                              | Value |
-| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ----- |
-| `postgresql.parameters.max_connections` | Determines the maximum number of concurrent connections to the database server. The default is typically 100 connections | `100` |
-| `quorum.minSyncReplicas`                | Minimum number of synchronous replicas that must acknowledge a transaction before it is considered committed.            | `0`   |
-| `quorum.maxSyncReplicas`                | Maximum number of synchronous replicas that can acknowledge a transaction (must be lower than the number of instances).  | `0`   |
-| `users`                                 | Users configuration                                                                                                      | `{}`  |
-| `databases`                             | Databases configuration                                                                                                  | `{}`  |
+| Name                                    | Description                                                                                                              | Type     | Value   |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ | -------- | ------- |
+| `external`                              | Enable external access from outside the cluster                                                                          | `bool`   | `false` |
+| `size`                                  | Persistent Volume size                                                                                                   | `string` | `10Gi`  |
+| `replicas`                              | Number of Postgres replicas                                                                                              | `int`    | `2`     |
+| `storageClass`                          | StorageClass used to store the data                                                                                      | `string` | ``      |
```

Comment:
Parameters don't appear in the same order as they are in values.yaml. It's important to order variables by importance and maintain the same order across all apps.

Reply:
Fixed.